src/quantity/__init__.py

```diff
@@ -3,7 +3,13 @@
 Copyright (c) 2024 Astropy Developers. All rights reserved.
 """
 
+from . import api
 from ._src import Quantity
 from .version import version as __version__  # noqa: F401
 
-__all__ = ["Quantity"]
+__all__ = [
+    # modules
```

Comment:
If we expose it, I think we should add tests...

Reply:
added!